packages/v4/src/content/design-guidelines/usage-and-behavior/forms/forms.md

```diff
@@ -5,254 +5,280 @@ hideTOC: true
+#### Top-aligned labels
+![Example of label aligned to the top of the input box](./img/top-labels.png)
 
-To accommodate mobile viewports, left-aligned labels should responsively adjust to a top-aligned layout.
+Use top-aligned labels by default. Top-aligned labels provide a consistent left edge and close proximity between label and input. Top-aligned labels also offer the most horizontal area for long labels.
 
-**Labeling for checkboxes and radio buttons**
+#### Left-aligned labels
+![Example of label aligned to the left of the input box](./img/left-labels.png)
 
-Checkboxes and radio buttons have a different labeling convention, where the labels should be aligned to the right of the input control.
 
-### Indicating required fields
-A required field should be indicated with an asterisk ( * ) to the right of the field label.
 
-![Required field image](./img/required-field.png)
 
-If **all fields** on a form are required, do not use an asterisk for every field. Instead, provide a message at the top of the form stating, "All fields are required." If all fields are optional, the message should state, "All fields are optional."
+Left-aligned labels are another available form option. They are useful for layouts with more horizontal space than vertical. However, as a general rule, they should be avoided in favor of top-aligned labels.
 
-![All required](./img/all-required.png)
+To accommodate mobile viewports, left-aligned labels should responsively adjust to a top-aligned layout for smaller viewports.
 
-### Provide the right data input
-PatternFly provides a variety of input components that enable a user to make a selection. Each component was created to serve a specific use case.
+#### Checkbox and radio button labels
+Checkboxes and radio buttons have a different labeling convention, where the option labels should be aligned to the right of the input control. 
 
-**Checkboxes**
+![Example of checkboxes and radio buttons aligned to the right of the input control](./img/radio-check.png)
 
-Use checkboxes to enable users to select any number of items from a list. The user could select all of the available options, some of them, or none.
 
-You can also present users with a single checkbox to enable or disable an option or setting.
+#### General labeling guidelines
+* Use sentence-style capitalization for all text elements except: product names, proper nouns and acronyms. Sentence-style capitalizes only the first word of each sentence and proper nouns, like names.
+* Always provide a label for input components.
+* Labels should clearly state the required input.
+* **Do not** use a colon after the label name.
 
-If you have more than 5 options to present to the user, use a select list.
+### Required fields
+A required field should be indicated with an asterisk ( * ) to the right of the field label.
 
-**Radio buttons**
+![Example of field with required field indication](./img/required-field.png)
```

Comment:
In this image, the asterisk seems very far away from the label compared with the implementation here: https://www.patternfly.org/v4/documentation/react/components/form Is this what is coming from the Sketch library?

Reply:
Yes, took everything straight out of the sketch library!

Reply:
Perhaps this is also a fix we need to make in Sketch then. @gdoyle1 see my comment. Has this always been this way? Sketch and the code are different then. Which do we think is correct?